api/threem.py

```diff
@@ -603,12 +603,12 @@ class ThreeMCirculationSweep(IdentifierSweepMonitor):
     count the same event.  However it will greatly improve our current
     view of our 3M circulation, which is more important.
     """
-    def __init__(self, _db, testing=False, api=None):
```

Comment:
All this `testing` stuff was unnecessary, since you can just pass in a mock API if you want. I took it out of all these classes.